openwisp_monitoring/device/api/views.py

```diff
@@ -235,7 +235,10 @@ def _write(self, request, pk, time=None, current=False):
                 extra_values = {
                     'tx_bytes': self._calculate_increment(
                         ifname, 'tx_bytes', ifstats['tx_bytes']
-                    )
+                    ),
+                    'total_bytes': self._calculate_increment(
+                        ifname, 'total_bytes', ifstats['rx_bytes'] + ifstats['tx_bytes']
+                    ),
```

Comment:
We don't need to store the `total_bytes` in the influxdb database. We can calculate it when we retrieve the data from the database.

Reply:
Ohh Okay! I will remove it from here. Thanks 👍🏻